components/identity.oauth.uma.permission.endpoint/src/main/webapp/WEB-INF/beans.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Don't we need to add license here ?. I am not sure .. Just check

Reply:
Since this was auto generated need the license header to be added?